src/test/java/com/puppycrawl/tools/checkstyle/xpath/XpathMapperTest.java

```diff
@@ -991,7 +992,8 @@ public void testTextBlockByItsValue() throws Exception {
     @Test
     public void testQuerySingleLineCommentByCommentContent() throws Exception {
         final String xpath = "//SINGLE_LINE_COMMENT[./COMMENT_CONTENT[@text=' some comment\\n']]";
-        final RootNode rootNode = getRootNodeWithComments("InputXpathMapperSingleLineComment.java");
+        final RootNode rootNode =
+                getFirstTopLevelNodeWithComments("InputXpathMapperSingleLineComment.java");
```

Comment:
this looks like conflicting in names:
RootNode vs getFirstTopLevelNodeWithComments

what update will be if we return actulal root note from this method?

Reply:
I changed entire update to unit tests; I have reverted the changes in functionality to the helper methods and just get each node in each test, as needed.